api/services/SiteCreator.js

```diff
@@ -70,8 +70,8 @@ function checkGithubOrg({ user, owner }) {
     if (!federalistAuthorizedOrg) {
       throw {
         message: `Federalist can't confirm org permissions for '${owner}'. ` +
-          `Either '${owner}' hasn't approved access for Federalist or you aren't an org member. ` +
-          'Ensure you are an org member and ask an org owner to authorize Federalist for the organization.',
+        `Either '${owner}' hasn't approved access for Federalist or you aren't an org member. ` +
+        `Ensure you are an org member and ask an org owner to authorize Federalist for the organization.`,
```

Comment:
I think this will fail linting. Backticks are only used when there is an interpolated value (demarcated with `${...}`) in the string .

Reply:
Ugh. That seems like weird styling but ok.

Reply:
It's not only styling, but kind of a check to make sure you are using the correct string delimiter type as appropriate. I do get that it can look kinda weird when they're right next to each other.